llvm/lib/Target/Xtensa/XtensaTargetMachine.h

```diff
@@ -36,10 +37,15 @@ class XtensaTargetMachine : public LLVMTargetMachine {
                       std::optional<CodeModel::Model> CM, CodeGenOptLevel OL,
                       bool JIT);
 
+  const XtensaSubtarget *getSubtargetImpl() const { return &Subtarget; }
```

Comment:
This method should not exist. Subtarget is per-function.

Reply:
Fixed. I implemented "getSubtargetImpl()" as it is done in M68k backend.